test/common-tests/acbs-get-facility-by-identifier-api-tests.ts

```diff
@@ -0,0 +1,79 @@
+import { TIME_EXCEEDING_ACBS_TIMEOUT } from '@ukef-test/support/environment-variables';
+import nock from 'nock/types';
+import supertest from 'supertest';
+
+export const withAcbsGetFacilityServiceCommonTests = ({
+  givenTheRequestWouldOtherwiseSucceed,
+  requestToGetFacility,
+  makeRequest,
+}: {
+  givenTheRequestWouldOtherwiseSucceed: () => void;
+  requestToGetFacility: () => nock.Interceptor;
+  makeRequest: () => supertest.Test;
+}) => {
+  describe('Common ACBS get facility service tests', () => {
+    it('returns a 404 response if ACBS get facility returns a 400 response with the string "The facility not found"', async () => {
+      givenTheRequestWouldOtherwiseSucceed();
+      requestToGetFacility().reply(400, 'The facility not found');
+
+      const { status, body } = await makeRequest();
+
+      expect(status).toBe(404);
+      expect(body).toStrictEqual({
+        statusCode: 404,
+        message: 'Not found',
+      });
+    });
+
+    it('returns a 500 response if ACBS get facility returns a 400 response without the string "The facility not found"', async () => {
+      givenTheRequestWouldOtherwiseSucceed();
+      requestToGetFacility().reply(400, 'An error message from ACBS.');
+
+      const { status, body } = await makeRequest();
+
+      expect(status).toBe(500);
+      expect(body).toStrictEqual({
+        statusCode: 500,
+        message: 'Internal server error',
+      });
+    });
+
+    it('returns a 500 response if ACBS get facility responds with a 400 response that is not a string', async () => {
```

Comment:
I had a chat to @lmarrai-sw and this is intended behaviour for the get endpoint (but not PUT or POST where it returns a 400). Correct me if I'm wrong Luke! :)

Reply:
Yep this is ACBS returning a 400 from a _GET_ request: we've been returning a 500 in that case across all our endpoints, since this should never actually happen in real usage of the app as we are validating the parameters of the GET request ourselves